io.openems.backend.timedata.aggregatedinflux/readme.adoc

```diff
@@ -0,0 +1,122 @@
+= Timedata Aggregated InfluxDB
+
+OpenEMS Backend implementation for aggregated InfluxDB access.
+
+[NOTE]
+====
+Bundle is needed for highly optimized environments with
+thousands of connected edges only. If you do not need that functionality you can skip this chapter.
+====
+
+A large number of OpenEMS edges connected to a single backend leads to a performance bottleneck.
+The bottleneck is mainly due to the access to the database. This `Aggregated InfluxDB` bundle in
+combination with the xref:../io.openems.backend.timedata.influx.adoc[Timedata Influx]
+bundle can mitigate this bottleneck. This happens by writing all data as usual via xref:../io.openems.backend.timedata.influx.adoc[Timedata Influx] into
+an Influx database (`DB 1`). Furthermore, all "aggregated data" (see xref:timedata.adoc[edge datatypes]) are written via the `Aggregated InfluxDB` bundle to
+another database (`DB 2`) located on another server.
+Thus `DB 1` holds all data in high resolution, while `DB 2` only holds very few data (5-minute averages for power values
+ or only daily values for energy). Furthermore, a retention policy of e.g. 90 days is applied to `DB 2`.
+Thus `DB 1` provides relatively slow access to a huge amount of data
+and `DB 2` provides fast and responsive access to a relatively small database.
+
+Within OpenEMS backend access to both databases is managed by the
+https://github.com/OpenEMS/openems/blob/develop/io.openems.backend.core/src/io/openems/backend/core/timedatamanager/TimedataManagerImpl.java[TimedataManager].
+The TimedataManager writes edge relevant data to **all** Timedata providers,
+whereas it reads data only from the **first** Timedata provider,
+ which can deliver.
+Assuming correct configuration, TimedataManager first returns historical data
+from `DB 2` (fast and responsive).
+And only in a very few cases it gets the data from `DB 1`
+(probably slower).
+
+
```

Comment:
Should we also mention what availableSince is doing?
I think it could be confusing when queries wont get handled by the AggregatedInflux because of it.